pkg/skr/gcpnfsvolume/reconciler.go

```diff
@@ -42,7 +43,7 @@ func (r *Reconciler) newAction() composed.Action {
 		composed.IfElse(EmptyLocationPredicate(), loadScope, nil),
 		composed.ComposeActions(
 			"crGcpNfsVolumeValidateSpec",
-			validateIpRange, validateFileShareName, validateCapacity, validatePV, validatePVC, validateLocation),
+			validateIpRange, validatePV, validatePVC, validateLocation),
```

Comment:
If these would be each one line, it would be much easier to track the change history.